packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig

```diff
@@ -17,4 +17,4 @@
   },
 ] %}
 
-<Breadcrumbs items={{ items }} />
+<Breadcrumbs items={{ items }} />
```

Comment:
an empty line at the bottom is missing?

Reply:
empty line added

Reply:
Hmm, this has already happened in another PR… @pavelklibani do you have Editorconfig installed in your IDE?